src/BlazorUI/Bit.BlazorUI/Components/Notifications/SnackBar/BitSnackBar.scss

```
@@ -55,14 +56,14 @@
 }
 
 .bit-snb-ttl {
-    font-weight: 500;
-    font-size: spacing(2);
+    font-weight: $tg-fw-medium;
+    font-size: $tg-fs-md;
     word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)